scripts/build/utils/utils.js

```diff
@@ -18,7 +18,7 @@ function getRuleTS({ includePaths }) {
 
 function getRuleCSS({ styleLoader, localIdentName, includePaths, context }) {
     return {
-        test: /\.scss$/,
+        test: /\.s?css$/,
```

Comment:
It's not related to this particular line of code, but when I run `npm run storybook`, I see this error:
![image](https://user-images.githubusercontent.com/403250/75881663-d04e2200-5e1f-11ea-9e3b-d9230e8cd8d3.png)